_middle/rights-and-responsibilities.md

```diff
@@ -0,0 +1,62 @@
+
+
+## Responsibilities
+
+- Follow lab policy when EPFL regulations delegate setting such policy to advisors, such as using a private computer for work ([EPFL lex 6.1.3](https://www.epfl.ch/about/overview/wp-content/uploads/2019/09/6.1.3_d_ordinateur_prive_an.pdf)), remote working ([EPFL lex 4.1.8](https://www.epfl.ch/about/overview/wp-content/uploads/2022/02/LEX-4.1.8_EN.pdf)), and others.
+- Conduct research with integrity and following good scientific practice ([EPFL lex 3.3.2](https://www.epfl.ch/about/overview/wp-content/uploads/2019/09/3.3.2_principe_integrite_recherche_an.pdf)), including relevant directives such as the one on [human blood sampling](https://www.epfl.ch/about/overview/wp-content/uploads/2019/09/1.5.10_dir_prise_de_sang_en.pdf)
+- Maintain professional secrecy regarding private information you learn as part of your job ([LPers](https://www.fedlex.admin.ch/eli/cc/2001/123/fr) art. 22)
+- Report some kinds of crimes as a mandatory reporter, see below
+
+
+## Mandatory reporting
+
+All federal employees, including PhD students, are mandatory reporters for some forms of crimes. ([LPers](https://www.fedlex.admin.ch/eli/cc/2001/123/fr) art. 22a)
+
+These are the crimes that the state prosecutes "ex officio", i.e., upon learning of them even without a formal complaint.
+They include domestic violence regardless of marital status and sexuality, attack with a weapon, negligence resulting in grave injuries, child abuse, and destroying property as part of a public gathering or in large amounts.
+
+There are narrow exceptions when the offender is a family member or when reporting would be dangerous to the reporter or a family member. ([CPP](https://www.fedlex.admin.ch/eli/cc/2010/267/fr#art_168) art. 168, 169)
+
+
+## Taking action
+
+If your rights or those of someone you know are violated, speak up as soon as possible!
+
+While you can get support for yourself regardless of when the infraction happened, the statute of limitations is typically a few years. ([EPFL ordinance 2.4.0.2](https://www.epfl.ch/about/overview/wp-content/uploads/2019/09/2.4.0.2Disciplinary_Rules_Regulations_ang.pdf) art. 6)
+
+You can contact:
+- Your PhD representatives, for any issues: <https://www.epfl.ch/education/phd/programs/edoc-student-representatives/>
+- The EPFL "person of trust", an independent external mediator mandated by EPFL, for interpersonal issues such as mobbing and harassment: <https://www.epfl.ch/about/respect/trust-and-support-network/person-of-trust-and-mediation-service/>
+- The EPFL "administrative ombudsperson", or as a second resort the ETH Board ones, for interpersonal issues: <https://www.epfl.ch/about/respect/the-respect-compliance-office/ombudsperson/>
+- The EPFL "scientific ombudsperson" for cases of scientific misconduct: <https://www.epfl.ch/research/ethic-statement/research-misconduct/>
```

Comment:
I have no clue what the difference is between the "person of trust" and the "administrative ombudsperson"